kong/pdk/service/response.lua

```diff
@@ -297,7 +297,7 @@ local function new(pdk, major_version)
   -- @tparam[opt] string mimetype The MIME type of the response (if known).
   -- @tparam[opt] number max_args Sets a limit on the maximum number of (what?)
   -- that can be parsed.
-  -- @treturn string The raw buffered body
+  -- @treturn string The raw decoded body
```

Comment:
```suggestion
  -- @treturn string The decoded buffered body
```
maybe you meant to replace `raw` here instead of `buffered`?